lib/node/Creator.js

```diff
@@ -53,7 +53,7 @@ const NodeCreator = {
     if (exponent) {
       polyTerm = this.operator('^', [polyTerm, exponent]);
     }
-    if (coeff) {
+    if (coeff && coeff.value !== '1') {
```

Comment:
This breaks some tests. Considering just doing this check before calling this. What do you think @evykassirer